### PR TITLE
DOC-2900 Timed Exams Remain Hidden After Due Date

### PR DESCRIPTION
**Pull request by catong**
Current default behavior is that timed exams are hidden after learners complete and submit the exam, but become available again as soon as the exam due date has passed. This PR adds CA doc for the ability to specify that a timed exam remains hidden from learners even after the exam due date has passed. 

TNL-4366/DOC-2900

Reviewers:
- [x] Subject matter expert: @efischer19 
- [x] Doc team: @srpearce

### CODE REVIEW

File: en_us/shared/course_features/timed_exams.rst [hunk 1]
```diff
@@ -122,3 +122,32 @@ extra time to complete a timed exam.
...
+
+
+#. In Studio, select the **Configure** icon for the timed exam (subsection)
+   that you want to configure.
+
+   The **Settings** dialog box opens to the **Basic** tab.
+
+#. Select the **Advanced** tab.
+
+#. In the **Set as a Special Exam** section, make sure **Timed** is selected.
```
Comment (efischer19) [inline en_us/shared/course_features/timed_exams.rst:147]: Maybe add something here about how this setting only applies to `Timed` exams; the other types will keep their default behavior of being visible after the due date.

**Comment by efischer19** [comment]
One minor comment, otherwise 👍 from me

**Comment by srpearce** [comment]
👍

**Comment by catong** [comment]
Thanks @efischer19 I added a note.

**Comment by efischer19** [comment]
Looks great @catong!

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/1012

**Pull request merged**